components/identity-core/org.wso2.carbon.identity.base/src/main/java/org/wso2/carbon/identity/base/IdentityBaseUtil.java

```diff
@@ -29,6 +29,10 @@
 
 import java.util.Properties;
 
+/**
+ * @deprecated Since no usages detected anymore within identity components.
+ */
+@Deprecated
 public class IdentityBaseUtil {
```

Comment:
[IWADeploymentInterceptor](https://github.com/wso2-extensions/identity-carbon-auth-iwa/blob/master/components/org.wso2.carbon.identity.authenticator.iwa/src/main/java/org/wso2/carbon/identity/authenticator/iwa/util/IWADeploymentInterceptor.java#L33) and [IWAUIAuthenticator](https://github.com/wso2-extensions/identity-carbon-auth-iwa/blob/master/components/org.wso2.carbon.identity.authenticator.iwa.ui/src/main/java/org/wso2/carbon/identity/authenticator/iwa/ui/IWAUIAuthenticator.java#L32) has references.

Reply:
Thanks for pointing out, didn't capture it. Will revert.